pkg/model/iam/iam_builder.go

```diff
@@ -777,6 +789,12 @@ func addKubeRouterSrcDstCheckPermissions(p *Policy) {
 	)
 }
 
+func addKindnetSrcDstCheckPermissions(p *Policy) {
+	p.unconditionalAction.Insert(
+		"ec2:ModifyInstanceAttribute",
```

Comment:
Not a blocker, but this has security/performance implications IIRC.  Do we need this permission in all modes?  (e.g. if we're running IPv6, do we need it?)